src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,64 @@
+package core.basesyntax;
+
+import core.basesyntax.db.FruitTransaction;
+import core.basesyntax.service.DataConverter;
+import core.basesyntax.service.DeepCopy;
+import core.basesyntax.service.FruitList;
+import core.basesyntax.service.ReaderService;
+import core.basesyntax.service.ReportGenerator;
+import core.basesyntax.service.WriterService;
+import core.basesyntax.service.impl.DataConverterImpl;
+import core.basesyntax.service.impl.DeepCopyImpl;
+import core.basesyntax.service.impl.FruitListImpl;
+import core.basesyntax.service.impl.ReaderServiceImpl;
+import core.basesyntax.service.impl.ReportGeneratorImpl;
+import core.basesyntax.service.impl.WriterServiceImpl;
+import core.basesyntax.strategy.BalanceHandler;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.PurchaseHandler;
+import core.basesyntax.strategy.ReturnHandler;
+import core.basesyntax.strategy.SupplyHandler;
+import java.util.List;
+
+public class Main {
+    public static void main(String[] args) {
```

Comment:
Don't begin class or method implementation with an empty line. Remove the redundant empty line at the start of the class and the main method.